h/static/styles/app.scss

```diff
@@ -50,6 +50,24 @@ body {
   }
 }
 
+.create-account-banner {
```

Comment:
Styles for individual components should go in their own .scss files which are then imported at the top.